introduction.md

```diff
@@ -0,0 +1 @@
+Hello! I am here to teach people about GitHub.
```

Comment:
hello

Reply:
https://www.codeschool.com/learn/git

Reply:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.
https://www.codeschool.com/learn/git